scripts/push-vellum-workflows.ts

```diff
@@ -148,6 +148,9 @@ class VellumWorkflowPusher {
       const errorOutput = (error.stdout || error.stderr || error.message || '').toString();
       if (errorOutput.includes('SDK Version') && errorOutput.includes('does not match SDK version') && errorOutput.includes('within the container image')) {
         console.log(`🔄 Detected SDK version mismatch for ${workflowName}, attempting to push new image...`);
+        console.error(`SDK Version: ${errorOutput.includes('SDK Version')}`);
+        console.error(`does not match SDK version: ${errorOutput.includes('does not match SDK version')}`);
+        console.error(`within the container image: ${errorOutput.includes('within the container image')}`);
```

Comment:
I want these outside of the if check bc idk if this if check is firing correctly